classes/security/UserGroupDAO.inc.php

```diff
@@ -399,12 +399,12 @@ function getContextUsersCount($contextId, $userGroupId = null, $roleId = null) {
 			'SELECT	COUNT(DISTINCT(uug.user_id)) AS row_count
 			FROM	user_groups ug
 				JOIN user_user_groups uug ON ug.user_group_id = uug.user_group_id
-			WHERE	context_id = ?' .
+			WHERE	ug.context_id = ?' .
 				($userGroupId?' AND ug.user_group_id = ?':'') .
 				($roleId?' AND ug.role_id = ?':''),
 			$params
 		);
-		$row = $result->current();
+		$row = (array) $result->current();
```

Comment:
This is fine (and I'll merge it as is), but I'm starting to prefer the object syntax to the array syntax. In this case, instead of:
```
$row = (array) $result->current();
return $row ? $row['row_count'] : 0;
```
...you would write...
```
$row = $result->current();
return $row ? $row->row_count : 0;
```